### PR TITLE
fix #983

### LINKED ISSUE #983: [Utilities] Add a warning message for NIfTI to DICOM conversion
**Is your feature request related to a problem? Please describe.**
DICOM images are tricky and users need to know what they need to do.

**Describe the solution you'd like**
Since NIfTI to DICOM is already present, add a warning message.

### PR DESCRIPTION
**Pull request by sarthakpati**
Fixes issue #983 

## Proposed Changes
  - added message for NIfTI to DICOM conversion
  - unique feature names (when applicable)

### CODE REVIEW

File: src/applications/Utilities/src/Utilities.cxx [hunk 1]
```diff
@@ -255,6 +255,8 @@ int algorithmsRunner()
   }
   else if (requestedAlgorithm == Nifti2Dicom)
   {
+    std::cout << "!!! WARNING: " <<
+      "Trying to write DICOM from NIfTI is dependent on the fact that the reference DICOM and NIfTI are in the same physical space and describe the same organ type.\n";
```
Comment (ashishsingh18) [inline src/applications/Utilities/src/Utilities.cxx:259]: Should the message not be more explicit?

Comment (sarthakpati) [reply src/applications/Utilities/src/Utilities.cxx:259]: I was envisioning something a bit generic. We can make this more verbose/specific as needed in the future.

Comment (ashishsingh18) [reply src/applications/Utilities/src/Utilities.cxx:259]: It must say somewhere that the reference dicom should match with nifti. Without that the output is prone to error.

Comment (sarthakpati) [reply src/applications/Utilities/src/Utilities.cxx:259]: Actually, I would counter that **match with nifti** is prone to erroneous interpretation but **the same physical space and describe the same organ type** is not.

Comment (ashishsingh18) [reply src/applications/Utilities/src/Utilities.cxx:259]: "same dicom that produced the nifti" is more important than organ type. If this is met, physical dimension condition is automatically met. Same organ type can be t1, t2 or can be a completely different modality which will still produce erroneous result. I leave the wording to you.

**Pull request merged**